### DISCUSSION TITLE
403 Forbidden - GHCR

### DISCUSSION THREAD

**Discussion by RyanMBess** [community/community#36838] [Packages]
I've started learning Ansible Automation Platform and it's use of containers. I was able to push containers to [ghcr.io](https://ghcr.io/) but then it stopped working. I suspect I'm hitting up against some personal use limit but can't validate that. I can authenticate successfully but every time i try and push to ghcr i get the below. I went into the Package repo of my github account and deleted the old containers. Thoughts?

StatusCode: 403, <html><body><h1>403 Forbidden</h1>

Request forbidd...

**Comment by ossanna16** [comment]
Thank you for asking a good question @RyanMBess, I'm glad you found our community 🙂
